ngraph/test/visitors/op/binary_ops.hpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "gtest/gtest.h"
+#include "util/visitor.hpp"
+
+template <typename T, ngraph::element::Type_t ELEMENT_TYPE>
+class BinaryOperatorType
+{
+public:
+    using op_type = T;
+    static constexpr ngraph::element::Type_t element_type = ELEMENT_TYPE;
+};
+
+template <typename T>
+class BinaryOperatorVisitor : public testing::Test
+{
+};
+
+class BinaryOperatorTypeName
+{
+public:
+    template <typename T>
+    static std::string GetName(int)
+    {
+        using OP_Type = typename T::op_type;
+        constexpr ngraph::element::Type precision(T::element_type);
+        const ngraph::Node::type_info_t typeinfo = OP_Type::get_type_info_static();
+        std::string op_name{typeinfo.name};
+        op_name.append("_");
+        return (op_name.append(precision.get_type_name()));
```

Comment:
I possible that bellow example will use RVO.
```suggestion
        return std::string{typeinfo.name} + "_" + precision.get_type_name();
```

Reply:
Refactored just as you suggested. Thank you!